backend/db/database_test.go

```diff
@@ -14,15 +14,33 @@ var (
 		Developer:    "tester",
 		CreationDate: time.Date(1900, 1, 1, 0, 0, 0, 0, time.UTC),
 		Version:      "0.0.0",
-		Tags:         []string{"tag0", "tag1"},
+		Tags:         []string{"tag9", "tag10"},
 	}
 
 	game1 = models.Game{
 		Name:         "game1",
 		Developer:    "tester",
 		CreationDate: time.Date(1900, 1, 2, 0, 0, 0, 0, time.UTC),
 		Version:      "0.0.1",
-		Tags:         []string{"tag1", "tag2"},
+		Tags:         []string{"tag11", "tag12"},
```

Comment:
Nope

Reply:
this is old commit and the HEAD even after this has been merged into main so this was just trivial commits, the pull request is last two commits only which are new commit others are old that has already been merged